### PR TITLE
Change installation method order

### PR DESCRIPTION
**Pull request by ifdattic**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.4 |
| Fixed tickets |  |

### CODE REVIEW

File: components/debug/introduction.rst [hunk 1]
```diff
@@ -16,8 +16,8 @@ Installation
 
 You can install the component in many different ways:
 
-* Use the official Git repository (https://github.com/symfony/Debug);
 * :doc:`Install it via Composer </components/using_components>` (``symfony/debug`` on `Packagist`_).
+* Use the official Git repository (https://github.com/symfony/Debug);
```
Comment (xabbuh) [inline components/debug/introduction.rst]: You need to swap the full stop and the semicolon too.

**Comment by WouterJ** [comment]
Could you please redo this PR against the 2.3 branch? Since you based on a not maintained branch, I can't rebase it due to submodule mismatches...

**Comment by ifdattic** [comment]
I don't think it would be possible to PR against 2.3 branch as this page doesn't contain a 2.3 branch entry...

**Comment by WouterJ** [comment]
@ifdattic it did exist, but in another location: http://symfony.com/doc/2.3/components/debug.html

There seems to be an issue with these documentation moves and the version switcher.

**Comment by weaverryan** [comment]
Good catch @ifdattic! So I merged this into 2.5 and backported it to 2.3 at sha: 0afc6892a0420090a8124f8d785d36b8679b2cf1

So, we're good to go - thanks!

**Comment by ifdattic** [comment]
Haven't thought about it being in another file. 
Glad to help

**Pull request closed without merging**